content/en/news/releases/1.6.x/announcing-1.6/announcing-1.6.1/index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Announcing Istio 1.6.1
+linktitle: 1.6.1
+subtitle: Patch Release
+description: Istio 1.6.1 patch release.
+publishdate: 2020-06-03
+release: 1.6.1
+aliases:
+    - /news/announcing-1.6.1
+---
+
+This release contains bug fixes to improve robustness .... This release note describes
+what’s different between Istio 1.6.0 and Istio 1.6.1.
+
+{{< relnote >}}
+
+## Changes
+- **Fixed** support for Pod annotations to override mesh-wide Proxy settings
+- **Fixed** support for all Envoy filters supported in [Envoy's go-control-plane](https://github.com/envoyproxy/go-control-plane) ([Issue 23909](https://github.com/istio/istio/issues/23909))
+- **Fixed** handling of custom resource names for Gateways ([Issue 23303](https://github.com/istio/istio/issues/23303))
+- **Fixed** `istiod` fails to issue certificate to a remote cluster ([Issue 23879](https://github.com/istio/istio/issues/23879))
+- **Fixed** remote cluster's validation controller to check `istiod's` ready status endpoint ([Issue 23945](https://github.com/istio/istio/issues/23945))
+- **Fixed** `regexp` `StringMatches` so they can be compiled by `Go` and `Envoy` ([Issue 23436](https://github.com/istio/istio/issues/23436))
+- **Fixed** `istioctl analyze` to validate `networking.istio.io/v1beta1` resources ([Issue 24064](https://github.com/istio/istio/issues/24064))
+- **Fixed** typo of `istio` in `ControlZ` dashboard log ([Issue 24039](https://github.com/istio/istio/issues/24039))
+- **Fixed** tar name to directory translation ([Issue 23635](https://github.com/istio/istio/issues/23635))
+- **Improved** certificate management for multi-cluster and virtual machine setup from `samples/certs` directory to `install/tools/certs` directory
+- **Improved** `pilot-agent's` handling of client certificates when only a CA client certificate is present
```

Comment:
cc @deva26 Please review the phrasing of your fix. Thanks.

Reply:
Thanks, LGTM.